soroban-env-host/src/native_contract/token/contract.rs

```diff
@@ -33,14 +36,22 @@ pub trait TokenTrait {
     /// called by the create_token_from_asset host function for this reason.
     ///
     /// No admin will be set for the Native token, so any function that checks the admin
-    /// (burn, freeze, unfreeze, mint, set_admin) will always fail
+    /// (clawback, freeze, unfreeze, mint, set_admin) will always fail
     fn init_asset(e: &Host, asset_bytes: Bytes) -> Result<(), HostError>;
 
     fn nonce(e: &Host, id: Identifier) -> Result<i128, HostError>;
 
     fn allowance(e: &Host, from: Identifier, spender: Identifier) -> Result<i128, HostError>;
 
-    fn approve(
+    fn incr_allow(
```

Comment:
Just food for thought: given that we went with the signed integers, would it make sense to just have a single function that modifies allowance by either positive or negative value? Not sure about the name though...

Reply:
My concern with that is users could assume it operates as a set function instead of an increment/decrement. Maybe we could avoid this with a good name, but we'd need more than 10 characters for that.

Reply:
Yeah let's hold off for now.